pkg/eth/erc20/bridge_contract.go

```diff
@@ -0,0 +1,390 @@
+package erc20
+
+import (
+	"context"
+	"errors"
+	"math/big"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+
+	tfeth "github.com/threefoldfoundation/tfchain/pkg/eth"
+	"github.com/threefoldfoundation/tfchain/pkg/eth/erc20/contract"
+	tftypes "github.com/threefoldfoundation/tfchain/pkg/types"
+)
+
+var (
+	// OneToken is the exact value of one token
+	OneToken = new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
```

Comment:
Why is this still a thing, and why is it still at 18 @LeeSmet? Is there a reason we do not use the tfchain constant OneCoin?